docs/configuration-reference/platforms/aws.md

```diff
@@ -159,24 +181,32 @@ worker_type = var.custom_default_worker_type
 | `network_mtu`                 | CNI interface MTU. Use 8981 if using instances types with Jumbo frames.                                                                                                                    | 1480            | false    |
 | `host_cidr`                   | CIDR IPv4 range to assign to EC2 nodes.                                                                                                                                                    | "10.0.0.0/16"   | false    |
 | `pod_cidr`                    | CIDR IPv4 range to assign Kubernetes pods.                                                                                                                                                 | "10.2.0.0/16"   | false    |
 | `service_cidr`                | CIDR IPv4 range to assign Kubernetes services.                                                                                                                                             | "10.3.0.0/16"   | false    |
 | `cluster_domain_suffix`       | Cluster's DNS domain.                                                                                                                                                                      | "cluster.local" | false    |
 | `enable_reporting`            | Enables usage or analytics reporting to upstream.                                                                                                                                          | false           | false    |
 | `certs_validity_period_hours` | Validity of all the certificates in hours.                                                                                                                                                 | 8760            | false    |
+| `worker_pool`                 | Configuration block for worker pools. There can be more than one.                                                                                                                          | -               | true     |
+| `worker_pool.count`           | Number of workers in the worker pool. Can be changed afterwards to add or delete workers.                                                                                                  | -               | true     |
+| `worker_pool.instance_type`   | AWS instance type for worker nodes.                                                                                                                                                        | "t3.small"      | false    |
+| `worker_pool.ssh_pubkeys`     | SSH public keys for user `core`.                                                                                                                                                           | -               | false    |
```

Comment:
This is actually required on my testing:

Reply:
https://github.com/kinvolk/lokomotive/pull/151/files#diff-3546ddc4d81aac8eac9975e948fef18bR37

Reply:
oh yep, sorry my bad.

Reply:
done.